1_prep.R

```diff
@@ -62,11 +53,102 @@ p1 <- list(
+
+  # filter observed data to mo lakes
+  tar_target(p1_obs_mo_lakes,
+             read_feather(p1_merged_temp_data_daily_feather) %>%
+               filter(site_id %in% p1_nldas_site_ids)
+             ),
+
+  # list manual cooperator crosswalk files
+  tar_files(p1_obs_manual_xwalks,
+             list.files('1_prep/in/obs_review/spatial/csvs', full.names = TRUE)
+           ),
+
+  # create missing cooperator crosswalks and dam crosswalk
+  tar_target(p1_obs_coop_missing_xwalks,
+             create_missing_xwalk(file_in = p1_obs_manual_xwalks,
+                                   path_out = '1_prep/in/obs_review/spatial'),
+             format = 'file',
+             pattern = p1_obs_manual_xwalks),
+
+  # list all cooperator data crosswalks
+  tar_files(p1_obs_coop_xwalks,
+            c(p1_obs_coop_missing_xwalks,
+              list.files('1_prep/in/obs_review/spatial',
+                         full.names = TRUE)) %>%
+              .[str_detect(., 'rds')] %>%
+              .[!str_detect(., 'wqp')] %>%
+              .[!str_detect(., 'dam')] %>%
+              unique()
+  ),
+
+  # list all cooperator data files from `7a_temp_coop_munge/tmp` in `lake-temperature-model-prep`
+  tar_files(p1_obs_coop_tmp_data,
+            list.files('1_prep/in/obs_review/temp', full.names = TRUE)
+  ),
+
+  # brittle - hacky work around to make sure the temp data is
+  # correctly paired with each xwalk - based on manual inspection
+  tar_target(p1_obs_coop_files,
+             tibble(
+               temp_files = p1_obs_coop_tmp_data,
+               xwalk_files = p1_obs_coop_xwalks[c(1, 2, 4, 3, 6, 5)]
+               )
+             ),
+
+  # add spatial information back into the observed data for WQP sites
+  tar_target(p1_obs_wqp_repaired,
+             repair_wqp_data(data = p1_obs_mo_lakes,
+                              xwalk = '1_prep/in/obs_review/spatial/wqp_lake_temperature_sites_sf.rds')
+             ),
+
+  # add spatial information back into the observed data for coop sites
+  tar_target(p1_obs_coop_repaired,
+             repair_coop_data(tbl_row = p1_obs_coop_files,
+                               data = p1_obs_mo_lakes),
+             pattern = p1_obs_coop_files),
+
+  # combine both data sets
+  tar_target(
+    p1_obs_data_w_spatial,
+    bind_rows(p1_obs_wqp_repaired, p1_obs_coop_repaired)
+  ),
+
+  # set dam buffer distances - units = meters
+  tar_target(p1_dam_buffer,
+             c(5000, 10000, 15000)),
+
+  # subset data based on distance from the dam
+  tar_target(
+    p1_obs_buff_sf,
+    readRDS(p1_obs_coop_missing_xwalks[agrep('dam_sf',
+                                             p1_obs_coop_missing_xwalks)]) %>%
+      st_buffer(., dist = p1_dam_buffer) %>%
+      st_intersection(p1_obs_data_w_spatial, .) %>%
+      mutate(buffer_dist = p1_dam_buffer),
+    pattern = p1_dam_buffer,
+    iteration = 'list'
+  ),
+
+  # subdivide data based on `site_id`
+  tar_target(
+    p1_obs_buffer_from_dam,
```

Comment:
This target hasn't been used yet. It's similar to `p1_obs_rds`, but specific to each distance-from-the-dam data subset. It will be used to create the 

This target does need to be cleaned up. It's a file target that has been improperly named and ID'd in the pipeline.